hooks/useTokenBalance.tsx

```diff
@@ -0,0 +1,45 @@
+import { TokenListItem } from "@/hooks/useTokenList";
+import { BigNumber } from "@ethersproject/bignumber";
+import { readContracts } from "@wagmi/core";
+import useSWR from "swr";
+import { useAccount, useNetwork } from "wagmi";
+
+export const ERC20_ABI = [
+  "function balanceOf(address owner) view returns (uint256 balance)",
```

Comment:
per @mds1 we can use a wagmi import here:

```js
import { erc20ABI } from "@wagmi/core";
```

Reply:
I don't know why but using this abi causes some new type errors. Spent a little time trying to fix, but seems good enough and maybe an improvement we can make later. FYI the type issue was 

```
Type error: Conversion of type '{} & readonly unknown[]' to type 'BigNumber | null' may be a mistake 
because neither type sufficiently overlaps with the other. If this was intentional, convert the expre
ssion to 'unknown' first.

  31 |         return {
  32 |           ...token,
> 33 |           balance: data[i] as BigNumber | null,
     |                    ^
  34 |         };
  35 |       });
  36 |       return tokelistWithUserBalance;
```

Reply:
it's a nit, let's track in #44 ; all good!